lms/templates/bookmarks/bookmarks_button.underscore

```diff
@@ -0,0 +1,5 @@
+<label for="course-bookmarks-button" class="sr"><%= gettext("Course") %></label>
+<button type="button" class="bookmarks-button" id="course-bookmarks-button">
```

Comment:
@muhammad-ammar I found one interaction issue here.  From a screen reader user's perspective, when they click the Bookmarks button, it seems like nothing has changed.  The focus remains on the button.  They did not see the content in the main content change.  To solve this, all you have to do is move focus() to something focusable in the main content area.  It has to be at the very top though.  In this instance, there is a header there "[My Bookmarks](https://github.com/edx/edx-platform/pull/7896/files#diff-a54cdc6968e13cd1db629bf843cce31dR1)".  We have a similar issue with the Sequence navigation in courses.  I suggest adding the following just above that header text:
```
<div id="my-bookmarks" class="sr-is-focusable" tabindex="-1"></div>
``` 
This is an empty div that takes up no space and will not have a visible outline when focused is moved to it.  tabindex="-1" allows the element to receive focus *only* from a script, so its not in the default tab order.  When the Bookmarks button is pressed, move focus to #my-bookmarks.  Make sure to do this last.  It has to exist on the page before you can move focus to it.  After pressing the Bookmarks button, focus will be moved there.  The next keyboard TAB or arrowkey press will take them to the My Bookmarks heading and the content that follows.  This is very helpful for screen reader users too.